drake/systems/onlinePlanning/FunnelLibraries/sampledFiniteTimeReach_B0.m

```diff
@@ -0,0 +1,1506 @@
+function [V,rho,Phi] = sampledFiniteTimeReach_B0(sys,polyOrig,Vtraj0,G,R0,tv,ts,xtraj0,utraj,options,Phi,rho)
```

Comment:
why not "sampledFiniteTimeReachability"

and I think this should definitely be a method of PolynomialSystem, like all of it's counterparts, not FunnelLibraries.